smithy-aws-apigateway-openapi/src/main/java/software/amazon/smithy/aws/apigateway/openapi/AddDefaultConfigSettings.java

```diff
@@ -16,30 +16,37 @@
 package software.amazon.smithy.aws.apigateway.openapi;
 
 import java.util.List;
+import java.util.logging.Logger;
 import software.amazon.smithy.model.Model;
 import software.amazon.smithy.openapi.OpenApiConfig;
 
 /**
- * Disables OpenAPI and JSON Schema features not supported by API Gateway.
+ * Sets default config settings for API Gateway.
+ *
+ * <p>By default, this disables OpenAPI and JSON Schema features not
+ * supported by API Gateway.</p>
  *
  * <p>API Gateway does not allow characters like "_". API Gateway
  * doesn't support the "default" trait or `int32` or `int64` "format"
  * values.
  */
 final class AddDefaultConfigSettings implements ApiGatewayMapper {
+    private static final Logger LOGGER = Logger.getLogger(AddDefaultConfigSettings.class.getName());
+    private static final String DEFAULT_VERSION_MESSAGE = String.format("`apiGatewayDefault` configuration not set for"
```

Comment:
I was more thinking of making something like `ApiGatewayDefault DEFAULT_VERSION = ApiGatewayDefault.VERSION_2023_08_11;` and then referencing that from the non-static message template and as the default value to apply when none is set.